README.md

```diff
@@ -0,0 +1,39 @@
+# Enumerable
+
+This project contain class MyList and a module MyEnumerable. The module MyEnumerable is implementing a subset of the functionality of [Enumerable.](https://ruby-doc.org/core-3.0.0/Enumerable.html)
+
+## Built with
+
+- Ruby
+
+## Getting Started
+
+To start using this software you need to have Ruby installed in your machine
+
+## Authors
+
+👤 **Ismail courr**
+
+- GitHub: [@ismailco](https://github.com/ismailco)
+- Twitter: [@ismailcourr](https://twitter.com/ismailcourr)
+- LinkedIn: [ismailcourr](https://linkedin.com/in/ismailcourr)
+
+👤 **Nicholas Mutale**
+
+- GitHub: [@mutalenic](https://github.com/Mutalenic)
+- Twitter: [@nicomutale](https://twitter.com/nicomutale)
+- LinkedIn: [@nicomutale](https://www.linkedin.com/in/nicomutale/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Your MIT link does not work. Kindly fix it. You can create the MIT file and change your URL to match its location, just as we have on the [Microverse template](https://github.com/microverseinc/readme-template).